docs/resources/compute_keypair_v2.md

```diff
@@ -17,11 +17,14 @@ resource "opentelekomcloud_compute_keypair_v2" "test-keypair" {
 
 ## Argument Reference
 
+->
+When creating an SSH key, you only need to configure `name`. When importing an SSH key, you must configure `public_key`.
+
```

Comment:
Maybe put this note after the `public_key` definition just before the later note?